core/rawdb/freezer_table_test.go

```diff
@@ -579,7 +579,7 @@ func TestOffset(t *testing.T) {
 		// It should be fine to fetch 4,5,6
 		if got, err := f.Retrieve(4); err != nil {
 			t.Fatal(err)
-		} else if exp := getChunk(20, 0xbb); !bytes.Equal(got, exp) {
+		} else if exp := getChunk(16, 0xbb); !bytes.Equal(got, exp) {
```

Comment:
I think there exists a design issue(we never ever use the deletion). Now if the head items are deleted, the number will be saved in `offset` field in the first index entry.

However, we don't record the "start_offset" of the new head item.